src/ir/comment.rs

```diff
@@ -38,6 +38,38 @@ fn make_indent(indent: usize) -> String {
     iter::repeat(' ').take(indent * RUST_INDENTATION).collect()
 }
 
+/// Preprocesses comments containing brackets to prevent `unresolved link to ...`
+///
+/// Avoid processing expensive computations if no brackets are found
+fn preprocess_plain_brackets(line: &str) -> String {
```

Comment:
Let's return a `Cow<str>` instead and avoid cloning the string if there are no brackets.